pygmt/tests/data/RidgeTest.prj

```diff
@@ -0,0 +1 @@
+GEOGCS["GCS_WGS_1984",DATUM["D_WGS_1984",SPHEROID["WGS_1984",6378137.0,298.257223563]],PRIMEM["Greenwich",0.0],UNIT["Degree",0.0174532925199433]]
```

Comment:
Still not sure if storing the shapefile (shp/shx/dbf/prj) in the git history is a good idea. I'm thinking whether we should put it on `dvc`, or maybe add some custom code to [`pygmt.helpers.testing.download_test_data()`](https://github.com/GenericMappingTools/pygmt/blob/cd8dcefb1ded453cc4873552d169c0bac0e6043f/pygmt/helpers/testing.py#L147-L189) to download the shapefile using `urllib` or something.

Reply:
> Another option would be to convert RidgeTest.gmt to a shapefile (possibly we could cache the shapefile to speed up the new PyGMT test and [https://github.com/GenericMappingTools/gmt/blob/master/test/psxy/categorical.sh](https://github.com/GenericMappingTools/gmt/blob/master/test/psxy/categorical.sh?rgh-link-date=2022-01-07T18%3A21%3A13Z)):
> 
> ```
> test_data=`gmt which -Gl @RidgeTest.gmt`
> ogr2ogr -f "ESRI Shapefile" RidgeTest.shp $test_data
> ```

I'm considering the option from @meghanrjones again. I think it makes more sense to cache the shapefile in the GMT server, because they can also be used by GMT and GMT.jl.

Reply:
Opened a PR in https://github.com/GenericMappingTools/gmtserver-admin/pull/142.